src/Compilers/VisualBasic/Test/Semantic/IOperation/IOperationTests_IVariableDeclaration.vb

```diff
@@ -800,8 +819,9 @@ IVariableDeclarationStatement (1 declarations) (OperationKind.VariableDeclaratio
     Variables: Local_1: i1 As System.Int32
       Local_2: i2 As System.Int32
     Initializer: 
-      IInvalidExpression (OperationKind.InvalidExpression, Type: ?, IsInvalid) (Syntax: 'i1')
-        Children(0)
+      ILocalInitializer (OperationKind.LocalInitializer, IsInvalid) (Syntax: 'i1')
```

Comment:
>ILocalInitializer (OperationKind.LocalInitializer, IsInvalid) (Syntax: 'i1') [](start = 6, length = 76)

Syntax for this node and its child looks incorrect.

Reply:
Seems like same issue as above, invalid expression child of a variable initializer has wrong syntax.

---
In reply to: [143050459](https://github.com/dotnet/roslyn/pull/22546#discussion_r143050459) [](ancestors = 143050459)